app-misc/homeassistant-bin/files/homeassistant.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Home Assistant
+After=network-online.target
+
+[Service]
+Type=simple
+User=homeassistant
+ExecStart=/opt/homeassistant/bin/hass --log-file /var/log/homeassistant/homeassistant.log $HASS_OPTS
+EnvironmentFile=/etc/conf.d/homeassistant
```

Comment:
We should probably split this configuration file from the openrc one as well (see: https://wiki.gentoo.org/wiki/Project:Systemd/conf.d_files)